client/my-sites/theme/main.jsx

```diff
@@ -387,12 +353,57 @@ const ThemeSheet = React.createClass( {
 	},
 } );
 
+const WrappedThemeSheet = ( props ) => {
+	if ( ! props.isLoggedIn || props.selectedSite ) {
```

Comment:
Could maybe eliminate this branch by having an `enabled` prop on `<ThemesSiteSelectorModal/>`. Fine as is, though.